package.json

```diff
@@ -2,7 +2,7 @@
   "name": "cloud-iframes",
   "version": "0.1.0",
   "private": true,
-  "homepage": "/sso",
+  "homepage": ".",
```

Comment:
Should it be commited?

Reply:
Yes, this is discussed with @jacekkolasa